Sources/DeepLearning/Layer.swift

```diff
@@ -52,7 +52,10 @@ public extension Layer {
 /// between training and inference shares an instance of ModeRef so it doesn't
 /// need to be toggled or threaded through in more than one place.
 public class ModeRef {
```

Comment:
We'll need to rename `ModeRef` to something Swifty. Suggestion made in another PR.